src/inabox/inabox-viewport.js

```diff
@@ -283,6 +341,12 @@ export class ViewportBindingInabox {
 
   /** @override */
   getRootClientRectAsync() {
+    if (this.canInspectTopWindow_()) {
+      return new Promise(resolve => {
+        resolve(layoutRectFromDomRect(
+            this.win.frameElement./*OK*/getBoundingClientRect()));
+      });
+    }
```

Comment:
This needs to be gated by the experiment too, right?

Reply:
Done.